src/AwkwardArray.jl

```diff
@@ -2730,7 +2730,7 @@ function _horizontal(data::Any, limit_cols::Int)
             if occursin(r"^[A-Za-z_][A-Za-z_0-9]*$", key)
                 key_str = key * ": "
             else
-                key_str = repr(key) + ": "
```

Comment:
This would have been an error, wouldn't it? I guess my tinkering on the command-line (the only way pretty-printing is tested) didn't run into this.

I never quite understood why the string concatenation operator in Julia is `*`. I was thinking it might be a math thing—thinking of strings as monoids, concatenation is not commutative and `+` is usually reserved for commutative operations. But is that really it?

`+` for string concatenation [goes back to ALGOL 68](https://retrocomputing.stackexchange.com/a/20461/26313)...

Reply:
https://docs.julialang.org/en/v1/manual/strings/#man-concatenation

tldr: `+` is usually commutative, `*` is often not, String concatenation is not commutative. (this argument can be abstracted into the monad thing).

My personal take: we write `2b` to mean `2 * b`, not `2 + b`, so juxtapositioning is `*`